src/api/l_audio_microphone.c

```diff
@@ -18,7 +18,8 @@ static int l_lovrMicrophoneGetChannelCount(lua_State* L) {
 
 static int l_lovrMicrophoneGetData(lua_State* L) {
   Microphone* microphone = luax_checktype(L, 1, Microphone);
-  SoundData* soundData = lovrMicrophoneGetData(microphone);
+  size_t samples = luax_optfloat(L, 2, 0);
```

Comment:
It seems a little bit odd to use `luax_optfloat` to read a `size_t`.

Reply:
I don't know what I was thinking... fixed